daml-assistant/integration-tests/src/DA/Daml/Assistant/IntegrationTestUtils.hs

```diff
@@ -39,6 +41,12 @@ withSdkResource f =
                         (extractDir </> "daml" </> damlInstallerName)
                         ["install", "--install-assistant=yes", "--set-path=no", extractDir]
                     else callCommandSilent $ extractDir </> "install.sh"
+                -- We restrict the permissions of the DAML_HOME directory to make sure everything
```

Comment:
shouldn’t do any harm but we don’t test telemetry in the integration tests so this is not testing the code path we care about.

Reply:
yes, it's meant to detect future dependency on a writable .daml directory.